pyproject.toml

```diff
@@ -43,3 +44,14 @@ add-ignore = 'D200,D202,D210,D212,D415,D105'
 [tool.mypy]
 check_untyped_defs = true
 ignore_missing_imports = true
+
+[tool.ruff]
```

Comment:
Shouldn't `ruff` be added as a python dependency? Isn't it redundant with `pydocstyle, `flake8`, etc., which are still marked as dependencies?

Reply:
It is indeed, I'll change that